performance-comparisons/v9.8/config/deployer.json

```
@@ -0,0 +1,113 @@
+{
+"ClusterName": "dkhcomparison",
```

Same indentation problem here
```suggestion
  "ClusterName": "dkhcomparison",
```

---

Resolved with latest commit.